crates/typos-cli/Cargo.toml

```diff
@@ -26,6 +26,7 @@ pre-release-replacements = [
   {file="../../CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n## [Unreleased] - ReleaseDate\n", exactly=1},
   {file="../../CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[Unreleased]: https://github.com/crate-ci/typos/compare/{{tag_name}}...HEAD", exactly=1},
   {file="../../docker/Dockerfile", search="ARG VERSION=.*", replace="ARG VERSION={{version}}", min=1},
```

Comment:
Since this Dockerfile is only for the action, we should remove it and the replacement for it

Reply:
Fixed.